.github/workflows/push-registry-app-files-supabase.yaml

```
@@ -0,0 +1,54 @@
+    steps:
+      - uses: actions/checkout@v4.1.7
+        name: Checkout repo
+        with:
+          # https://github.com/actions/checkout/issues/266#issuecomment-638346893
+          fetch-depth: 0
+      - uses: pnpm/action-setup@v4.0.0
+        with:
+          version: 9.14.2
+      - name: Get pnpm store directory
+        id: pnpm-cache
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)